apps/gro/confirm-step-config.js

```diff
@@ -3,7 +3,7 @@
 module.exports = {
   modifiers: {
     'about-radio': (value, req) =>
-      req.translate(`pages.confirm.table.values['about-radio'][${value}]`)
+      req.translate(`fields['about-radio'].options[${value}].label`)
```

Comment:
with hof-controller change, you could do `fields[${this.name}].options[${value}].label`